shell/edit/namespace.vue

```diff
@@ -65,13 +65,6 @@ export default {
     isHarvester() {
       return this.$store.getters['currentProduct'].inStore === HARVESTER;
     },
-    extraColumns() {
```

Comment:
I deleted this property because I checked the namespace edit config form and confirmed that it is not being used.